packages/webapp/src/pages/Connection/List.tsx

```diff
@@ -20,59 +20,55 @@ import { useStore } from '../../store';
 
 const defaultFilter = ['all'];
 
+function truncateWithEllipsis(text: string) {
+    if (text.length <= 6) {
+        return text;
+    }
+    return text.slice(0, 3) + '...' + text.slice(-3);
+}
```

Comment:
I could reuse that actually, do you mind putting that in utils (eventually in a new file utils/string.ts to avoid cluttering the main file)